job_spec/INSAR_ISCE_TEST.yml

```diff
@@ -37,6 +37,13 @@ INSAR_ISCE_TEST:
           - HRRR
           - HRES
           - GMAO
+    esd_coherence_threshold:
+      api_schema:
+        description: Coherence threshold value, between 0 and 1, for points to include when performing ESD, or -1 for no ESD correction.
+        default: -1.0
+        type: number
+        minimum: -1.0
+        maximum: 1.0
```

Comment:
Technically, this should be a range of `0.` to `1.`, OR `-1.`, but I don't think OpenAPI spec can do that, nor do I think it's necessarily worth doing. @asjohnston-asf @jtherrmann do you know by chance?

Reply:
Not seeing a way to do that: https://swagger.io/docs/specification/data-models/data-types/#numbers